template/cordova/lib/MSBuildTools.js

```diff
@@ -44,8 +52,39 @@ MSBuildTools.prototype.buildProject = function(projFile, buildType, buildarch, o
             args.push('/p:' + key + '=' + otherConfigProperties[key]);
         });
     }
+    var that = this;
 
-    return spawn(path.join(this.path, 'msbuild'), [projFile].concat(args), { stdio: 'inherit' });
+    // Check if SDK required to build the respective platform is present. If not present, return with corresponding error, else call msbuild.
+    if (projFile.indexOf('CordovaApp.Phone.jsproj') > -1) {
```

Comment:
I implemented it earlier that way but that didn't really look any cleaner (still required 3 different function calls, 3 different error messages and spawn calls). I thought this would look more readable. Thoughts?